package.json

```diff
@@ -4,11 +4,12 @@
     "description": "Live cricket scores served at your terminal 🏏",
     "main": "index.js",
     "scripts": {
-        "start": "node ./dist/index.js",
+        "start": "nodemon --exec npm run babel",
+        "babel": "babel-node index.js",
```

Comment:
I guess we should use the build file that parcel creates...What do you think?

Reply:
For the bundle, I agree, but in development, it doesn't allow to watch files. We would have to create a server using parcel config (which also uses babel config for esnext syntax and es6 modules if we want to use that in future). You could commit that on top of this branch. Other wise, I think this works fine right now.

Reply:
Yes, watching files is not really needed as of now. But, using the bundle gives some error during the build process. I guess we should go ahead with this for now and come back later for the bundling part. :+1: :rocket: 

Thanks!